api/src/main/java/com/microsoft/gctoolkit/io/GCLogFile.java

```diff
@@ -38,6 +40,15 @@ protected GCLogFile(Path path, boolean unifiedFormat) {
      */
     public boolean isUnifiedFormat() { return unifiedFormat; }
 
+    /**
+     *
+     * @return
+     */
+
+    public LoggingDiary diary() {
```

Comment:
I'm guessing the intent here is to have this method do the diarization? 

javadoc is going to barf if you don't provide values for the tags.